dags/resources/stages/parse/table_definitions/dodo/DODOV2Proxy_call_createDODOPrivatePool.json

```diff
@@ -0,0 +1,118 @@
+{
+    "parser": {
+        "abi": {
+            "inputs": [
+                {
+                    "internalType": "address",
+                    "name": "baseToken",
+                    "type": "address"
+                },
+                {
+                    "internalType": "address",
+                    "name": "quoteToken",
+                    "type": "address"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "baseInAmount",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "quoteInAmount",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "lpFeeRate",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "i",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "k",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "bool",
+                    "name": "isOpenTwap",
+                    "type": "bool"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "deadLine",
+                    "type": "uint256"
+                }
+            ],
+            "name": "createDODOPrivatePool",
+            "outputs": [
+                {
+                    "internalType": "address",
+                    "name": "newPrivatePool",
+                    "type": "address"
+                }
+            ],
+            "stateMutability": "payable",
+            "type": "function"
+        },
+        "contract_address": "SELECT '0xa356867fdcea8e71aeaf87805808803806231fdc' UNION ALL SELECT '0x9ae501385bc7996a2a4a1fbb00c8d3820611bcb5'",
```

Comment:
@ememet This line does not pass the syntax [test](https://github.com/blockchain-etl/ethereum-etl-airflow/actions/runs/4473432844/jobs/7860808619?pr=541). Can try using `UNNEST` like [here](https://github.com/blockchain-etl/ethereum-etl-airflow/blob/3dc3298f260d8fdcc229dddab0f2138a97d55661/dags/resources/stages/parse/table_definitions/niftygateway/BuilderShop_call_createNewBuilderInstance.json#L42)